packages/@aws-cdk/aws-cloudformation/package.json

```diff
@@ -66,7 +66,7 @@
     "@aws-cdk/assert": "^1.9.0",
     "@aws-cdk/aws-events": "^1.9.0",
     "@aws-cdk/aws-ssm": "^1.9.0",
-    "@types/aws-lambda": "^8.10.31",
+    "@types/aws-lambda": "^8.10.32",
```

Comment:
I think that this dependency is no longer used in this package, was used for the `AwsCustomResource`.

Reply:
Does look like there's a public API using this - https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cloudformation/lib/custom-resource.ts#L17

Reply:
Oops, my bad. Mistook `@types/aws-lambda` to `@aws-cdk/aws-lambda`